IPython/terminal/interactiveshell.py

```diff
@@ -424,21 +424,10 @@ def prompt_for_code(self):
                 **self._extra_prompt_options())
         return text
 
-    def enable_win_unicode_console(self):
-        if sys.version_info >= (3, 6):
-            # Since PEP 528, Python uses the unicode APIs for the Windows
-            # console by default, so WUC shouldn't be needed.
-            return
-
-        import win_unicode_console
-        win_unicode_console.enable()
-
```

Comment:
I would prefer to have this function as a "Do nothing", but emit a warning it is going to be removed in subsequent versions; to give some time to potential user to stop using it. 

something along:

```
def ...
    import warn
    warn("`enable_win_unicode_console` is deprecated since IPython 7.10, does not do anything and will be removed in the future", DeprecationWarning, stacklevel=2)

```

Reply:
I have added the deprecation warning withthis commit 747cdeb